services/cd-service/pkg/repository/overview_cache.go

```diff
@@ -0,0 +1,420 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the Expat(MIT) License as published by
+the Free Software Foundation.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+
+You should have received a copy of the MIT License
+along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+Copyright freiheit.com*/
+/*
+This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the Expat(MIT) License as published by
+the Free Software Foundation.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+
+You should have received a copy of the MIT License
+along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+Copyright freiheit.com
+*/
+package repository
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+
+	api "github.com/freiheit-com/kuberpult/pkg/api/v1"
+	"github.com/freiheit-com/kuberpult/pkg/grpc"
+	"github.com/freiheit-com/kuberpult/pkg/mapper"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+func UpdateOverviewDB(
+	ctx context.Context,
+	state *State,
+	transaction *sql.Tx,
+) error {
+	var rev string
+	if state.Commit != nil {
+		rev = state.Commit.Id().String()
+	}
+	result := &api.GetOverviewResponse{
+		Branch:            "",
+		ManifestRepoUrl:   "",
+		Applications:      map[string]*api.Application{},
+		EnvironmentGroups: []*api.EnvironmentGroup{},
+		GitRevision:       rev,
+	}
+	latestOverviewRow, err := state.DBHandler.ReadLatestOverviewCache(ctx, transaction)
+	if err != nil {
+		return err
+	}
+
+	if latestOverviewRow != nil {
+		err = json.Unmarshal([]byte(latestOverviewRow.Blob), result)
```

Comment:
`Unmarshal` (and `Marshal`) should be part of the function in db.go for consistency.